src/main.rs

```diff
@@ -110,7 +110,7 @@ async fn main() {
         let world = TypstSystemWorld::new(CompileOpts {
             root_dir: root.clone(),
             font_paths: arguments.font_paths.clone(),
-            with_embedded_fonts: get_embedded_fonts(),
+            with_embedded_fonts: typst_assets::fonts().map(Cow::Borrowed).collect(),
```

Comment:
`embed-fonts` is not valid anymore if you do that. People will always have to run preview with embedded fonts. Either remove feature `embed-fonts = ["typst-assets/fonts"]` and always enable "typst-assets/fonts", or revert this change.

Reply:
I think I may mistakely think of it. there is no problem.

Reply:
https://github.com/typst/typst-assets/blob/main/src/lib.rs#L65